app/schemas/selectors/cinematic.js

```diff
@@ -321,3 +327,15 @@ export const getBackgroundObject = compose(triggers, backgroundObject)
  * @returns {number|undefined} the delay before removing the background object.
  */
 export const getClearBackgroundObject = compose(triggers, clearBackgroundObject)
+
+/**
+ * @param {DialogNode} dialogNode
+ * @returns {number} defaults return of 1000
+ */
+export const getTextAnimationLength = textAnimationLength
```

Comment:
What is the difference between this and `textAnimationLength` above?

Reply:
I've been using a hideous convention where I export the `get<selector>` and then have a private `<selector>`. So in this case `TextAnimationLength` is private and would do the logic. And then I export a thin wrapper that either targets the DialogNode or the Shot.
From your great comments I realize it's too much and just confusing. Especially when the export is doing nothing more than re-exporting.